src/ray/raylet/worker_pool.h

```diff
@@ -431,6 +442,8 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
     std::unordered_set<TaskID> tasks_with_dedicated_workers;
```

Comment:
Do we still need the concept of "decicated" workers?

Reply:
Yes, decicated workers are used for the actor creation tasks with DynamicWorkerOptions. When we pop a idle worker, we need this to distinguish.